src/DssSpell.sol

```diff
@@ -49,82 +141,347 @@ contract DssSpellAction is DssAction {
+        // dist: 0xa3Ee378BdD0b7DD403cEd3a0A65B2B389A2eaB7e
+        // period: 7 days - 1 hours
+        VestedRewardsDistributionJobLike(CRON_REWARDS_DIST_JOB).set(REWARDS_DIST_LSSKY_SPK, 7 days - 1 hours);
 
-        // Reduce Global Debt Ceiling to account for this change
-        // Note: This includes all offboarded ilks above as well
-        DssExecLib.decreaseGlobalDebtCeiling(globalLineReduction / RAD);
+        // Add REWARDS_LSSKY_SPK farm to the Lockstake Engine:
+        // farm: 0x99cBC0e4E6427F6939536eD24d1275B95ff77404
+        LockstakeEngineLike(LOCKSTAKE_ENGINE).addFarm(REWARDS_LSSKY_SPK);
+
+        // Add REWARDS_LSSKY_SPK and REWARDS_DIST_LSSKY_SPK to the Chainlog:
+        // key: "REWARDS_LSSKY_SPK"
+        // addr: 0x99cBC0e4E6427F6939536eD24d1275B95ff77404
+        // key: "REWARDS_DIST_LSSKY_SPK"
+        // addr: 0xa3Ee378BdD0b7DD403cEd3a0A65B2B389A2eaB7e
+        DssExecLib.setChangelogAddress("REWARDS_LSSKY_SPK", REWARDS_LSSKY_SPK);
+        DssExecLib.setChangelogAddress("REWARDS_DIST_LSSKY_SPK", REWARDS_DIST_LSSKY_SPK);
+
+        // ---------- Convert MKR balance of the PauseProxy to SKY ----------
+        // Forum: https://forum.sky.money/t/phase-3-mkr-to-sky-migration-items-june-26-spell/26710
+        // Atlas: https://sky-atlas.powerhouse.io/A.4.1.2.1.4.2.3_Upgrade_MKR_In_Pause_Proxy_To_SKY/1f1f2ff0-8d73-8064-ab0e-d51c96127c19|b341f4c0b83472dc1f9e1a3b
+
+        // Note: get unpaid MKR for MCD_VEST_MKR_TREASURY ids 9, 18, 24, 35, 37, and 39
+        uint256 unpaidMkr = VestAbstract(MCD_VEST_MKR_TREASURY).unpaid(9) +
+            VestAbstract(MCD_VEST_MKR_TREASURY).unpaid(18) +
+            VestAbstract(MCD_VEST_MKR_TREASURY).unpaid(24) +
+            VestAbstract(MCD_VEST_MKR_TREASURY).unpaid(35) +
+            VestAbstract(MCD_VEST_MKR_TREASURY).unpaid(37) +
+            VestAbstract(MCD_VEST_MKR_TREASURY).unpaid(39);
+
+        // Note: approve MKR_SKY to spend MKR balance of the PauseProxy
+        ERC20Like(MKR).approve(MKR_SKY, ERC20Like(MKR).balanceOf(address(this)) - unpaidMkr);
+
+        // Call mkrToSky() on MKR_SKY with the MKR balance of the PauseProxy minus the unpaid() MKR for MCD_VEST_MKR_TREASURY ids 9, 18, 24, 35, 37, and 39
+        MkrSkyLike(MKR_SKY).mkrToSky(address(this), ERC20Like(MKR).balanceOf(address(this)) - unpaidMkr);
+
+        // ---------- Disable MKR_SKY_LEGACY Converter ----------
+        // Forum: https://forum.sky.money/t/phase-3-mkr-to-sky-migration-items-june-26-spell/26710
+        // Atlas: https://sky-atlas.powerhouse.io/A.4.1.2.2.4.2.1_Disabling_Legacy_Conversion_Contract/210f2ff0-8d73-80d4-a983-f9217c9b244a|b341f4c0b834477b310e7381
+
+        // Note: load DssInstance from chainlog
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
+
+        // Call disableOldConverterMkrSky() to deactive the legacy converter (https://github.com/sky-ecosystem/sky/pull/21/files#diff-f6cbf09833eed835c52b0a1c5be7dd9e84213d278c958843725af6a77faa77d4R69-R75)
```

Comment:
```suggestion
        // Call disableOldConverterMkrSky() to deactivate the legacy converter (https://github.com/sky-ecosystem/sky/pull/21/files#diff-f6cbf09833eed835c52b0a1c5be7dd9e84213d278c958843725af6a77faa77d4R69-R75)
```
Small typo here, although it comes from the exec sheet, so not sure if we should fix it or keep it the same.

Reply:
Usually it is preferable to keep the instructions from the sheet verbatim.
We should probably flag this to Gov Facilitators.
Not sure if the Exec Doc has the same issue. Will check.